docs/manpage.rst

```diff
@@ -364,6 +368,8 @@ If no node can be selected, the test will be marked as a failure with an appropr
      It is therefore possible that the number of tasks assigned does not correspond to the actual idle nodes.
 
      This is the default policy.
+
+   - ``maint``: Flexible tests will be assigned as many tasks as needed in order to span over the nodes of the node pool which are currently under maintenance.
```

Comment:
You should be more precise by mentioning the actual state that the nodes will be in.